xedocs/xedocs.py

```diff
@@ -127,6 +127,20 @@ def xedocs_protocol(name, version="ONLINE", sort=None, attr=None, **kwargs):
             return docs[0]
 
         return docs
+    
+    @URLConfig.register('docs-to-array')
+    def docs_to_array(docs: list):
+        import numpy as np
+        
+        if not isinstance(docs, list):
+            raise TypeError(f'The docs-to-array protocol expects a list but recieved a {type(docs)} instead')
+        result = np.array([])
+        
+        if isinstance(docs, list):
```

Comment:
this is also redundent, we already checked that this is a list. if its not a list we would have raised an error and would not reach this line of code.
```suggestion
```